package.json

```diff
@@ -24,14 +24,18 @@
     "dev": "lerna run --parallel --scope @firebase/* --scope firebase dev",
     "build": "lerna run --scope @firebase/* --scope firebase build",
     "build:changed": "ts-node-script scripts/ci-test/build_changed.ts",
+    "build:ci:auth": "lerna run build:deps --scope '@firebase/auth*'",
+    "build:ci:firestore": "lerna run build:deps --scope '@firebase/firestore*'",
     "release:prepare": "lerna run --scope @firebase/* add-compat-overloads && lerna run --scope @firebase/* typings:public",
     "link:packages": "lerna exec --scope @firebase/* --scope firebase -- yarn link",
     "stage:packages": "./scripts/prepublish.sh",
     "repl": "node tools/repl.js",
     "release": "ts-node-script scripts/release/cli.ts",
     "pretest": "node tools/pretest.js",
     "test": "lerna run --ignore firebase-messaging-integration-test --concurrency 4 --stream test",
-    "test:ci": "lerna run --ignore firebase-messaging-integration-test --concurrency 4 test:ci",
+    "test:ci": "lerna run --ignore '{firebase-messaging-integration-test,@firebase/auth*,@firebase/firestore*}' --concurrency 4 test:ci",
+    "test:ci:auth": "lerna run  --concurrency 4 test:ci --scope '@firebase/auth*'",
```

Comment:
We have a lot of scripts in package.json, instead of making new ones, can we just call lerna directly in the workflow? Should be able to call `yarn lerna run  --concurrency 4 test:ci --scope '@firebase/auth*'` in the workflow file.

Reply:
Done.